core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java

```diff
@@ -642,6 +656,17 @@ public NativeImageInvokerInfo build() {
                     enableHttpsUrlHandler = true;
                 }
 
+                List<String> featuresList = new ArrayList<>(graalVmFeatures == null ? 3 : graalVmFeatures.size() + 3);
+                featuresList.add(NativeImageFeatureStep.GRAAL_FEATURE);
+                featuresList.add(ResourcesFeature.class.getName());
+                featuresList.add(DisableLoggingFeature.class.getName());
```

Comment:
Can we have them produced elsewhere? IMHO It would be much better if this part could just consume them

Reply:
Done